File: python/prism_sdk/local_audit_integrity_inference.py
"""Sweep P32 local inference audit-integrity feature F01."""
from .audit_integrity_support import AuditRequest4,AuditCard7,AuditIntegrityError,manifest,qualify
FEATURE_ID="AFA-sweep-P32-F01";CONTRACT_VERSION="sweep-local_audit_integrity_inference/1.0"
def local_audit_integrity_inference_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="local",mode="inference")
def qualify_local_audit_integrity_inference(request:AuditRequest4)->AuditCard7:return qualify(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="local",mode="inference")

File: crates/sweep/src/local_audit_integrity_inference.rs
//! Sweep P32 local inference audit-integrity feature.
use super::audit_integrity_support::{manifest,qualify,AuditCard7,AuditIntegrityError,AuditRequest4};
pub const FEATURE_ID:&str="AFA-sweep-P32-F01";pub const CONTRACT_VERSION:&str="sweep-local_audit_integrity_inference/1.0";
pub fn local_audit_integrity_inference_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"local","inference")}
pub fn qualify_local_audit_integrity_inference(request:&AuditRequest4)->Result<AuditCard7,AuditIntegrityError>{qualify(request,FEATURE_ID,CONTRACT_VERSION,"local","inference")}
